src/main/java/Duke.java

```diff
@@ -1,10 +1,99 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
+        welcomeMessage();
+
+        Scanner sc = new Scanner(System.in);
+        List<Task> tasks = new ArrayList<Task>();
+        Boolean exitProgram = false;
+
+        while (!exitProgram) {
+            String input = sc.nextLine();
+            String[] inputArray = input.split(" ", 2);
+
+            System.out.println("____________________________________________________________");
```

Comment:
Would it be better to put this in a method (for example displayLine()) instead of a magic string?

Reply:
Thats a great idea. Will implement it into my code 👍